src/bin.js

```diff
@@ -2,4 +2,6 @@
 
 'use strict'
 
-require('./')()
+// First param is the target version
+// Second param is the target platform
+require('./')(process.argv[2], process.argv[3], function () {})
```

Comment:
Can we make that 'later' more explicit? With a TODO of what is desired for example.